apps/unsharp/unsharp_generator.cpp

```diff
@@ -62,14 +62,18 @@ class Unsharp : public Halide::Generator<Unsharp> {
 
         // Schedule
         if (!auto_schedule) {
+            // Mac Minis have Intel GPUs that don't have enough memory to handle 32x32
```

Comment:
Nit: "Intel Mac Minis have GPUs that require tile sizes smaller than 32x32 for this pipeline" or similar.  It's # of registers, not memory that's the issue here.

Reply:
Fixed.